unit-tests/engine_test.py

```diff
@@ -18,7 +18,7 @@
 
 class _StyleHarness(Style):
     def __init__(self) -> None:
-        super(_StyleHarness, self).__init__([])
+        super(_StyleHarness, self).__init__()
```

Comment:
While still valid, `super().__init__()` is preferred in Python 3. This same pattern appears in LanguageHarness in source_test.py

Reply:
Quite correct. Obviously a leftover from when we targeted Python2.